defaults/config.js

```diff
@@ -148,6 +148,19 @@ module.exports = {
 	// @default  null
 	webirc: null,
 
+	//
+	// Message logging
+	// Logging is also controlled per user individually (logs variable)
+	//
+	// none: Do not store any logs
+	// text: Text file per network/channel in user folder
+	// sqlite: Messages are stored in SQLite, this allows them to be reloaded on server restart
+	//
+	// @type     string
+	// @default  "text"
+	//
+	messageStorage: "text",
```

Comment:
Could this be an array? Allowing someone to specify ["sqlite", "text"] to get the advantage of reloading messages, but also be able to view text logs?

Reply:
I don't really want to do that because there's no storage interface yet, which I don't plan to add in this PR either.

This could be improved upon in a future PR.

Reply:
I'm not sure the disadvantage of just allowing an array just now, and doing both if thye are both in the array? Sure, it'll need to be expanded and tidied up in other stuff (I'm not suggesting doing the storage interface just now) but your existing logic just needs to change to equal or .includes and change the `else if` to just `if` when setting up logic. Am I missing something?